2_SERIES_UPGRADE.md

```diff
@@ -22,6 +22,14 @@ simply change this too:
 import quandl as Quandl
 ```
 
+Additionally if you were relying on a saved version of your `api key` _(authtoken)_ in your scripts please note that this functionality has been dropped from the existing package. To continue to load your api key from the filesystem we recommend you manually load the key and set it via the api config `quandl.ApiConfig.api_key`. Some example code that will do this is:
```

Comment:
"Manually load" is vague here, maybe we can drop it? To mean that means to actually open up the file in an editor to find my api key, and then copy and paste it into my code?

Also user's might know actually know where the file is, maybe we should reference the old README? https://github.com/quandl/quandl-python#usage